src/objects.h

```diff
@@ -10381,10 +10373,6 @@ class Float32x4: public JSObject {
   // Helpers.
   static const int kLanes = 4;
   inline float getAt(int index);
```

Comment:
In my comment https://github.com/crosswalk-project/v8-crosswalk/pull/118#discussion_r42583925, I made two suggestions 1) rename `getAt` to `getLane` 2) remove `x()/y()/z()/z()`. You did 2). How about 1)? Do you agree with that?